isisd/isis_circuit.c

```diff
@@ -1424,10 +1424,6 @@ static int isis_interface_config_write(struct vty *vty)
 					circuit->passwd.passwd);
 				write++;
 			}
-			if (circuit->bfd_config.enabled) {
```

Comment:
Is this printed by `cli_show_ip_isis_bfd_monitoring()` already?

Reply:
This code is called only for fabricd which is not YANGified.